cmake/config.cmake

```diff
@@ -41,9 +41,9 @@ set(OMR_PORT ON CACHE BOOL "Enable portability library")
 set(OMR_OMRSIG ON CACHE BOOL "Enable the OMR signal compatibility library")
 set(OMR_THREAD ON CACHE BOOL "Enable thread library")
 
-set(OMR_COMPILER OFF CACHE BOOL "Enable the Compiler")
-set(OMR_JITBUILDER OFF CACHE BOOL "Enable building JitBuilder")
-set(OMR_TEST_COMPILER OFF CACHE BOOL "Enable building the test compiler")
+set(OMR_COMPILER ON CACHE BOOL "Enable the Compiler")
+set(OMR_JITBUILDER ON CACHE BOOL "Enable building JitBuilder")
+set(OMR_TEST_COMPILER ON CACHE BOOL "Enable building the test compiler")
```

Comment:
Why is this being changed?

Reply:
Sorry, that was a careless mistake on my part.